JS/js-functions_dom-manipulation/js/index.js

```diff
@@ -15,24 +15,40 @@ redButton.addEventListener("click", () => {
   // use your functions to
   // - remove all classes
   // - then add the class name: "red"
+  removeAllColors();
+  addColor("red");
 });
 
 blueButton.addEventListener("click", () => {
   // use your functions to
   // - remove all classes
   // - then add the class name: "blue"
+  removeAllColors();
+  addColor("blue");
 });
 
 greenButton.addEventListener("click", () => {
   // use your functions to
   // - remove all classes
   // - then add the class name: "green"
+  removeAllColors();
+  addColor("green");
 });
 
 grayButton.addEventListener("click", () => {
   // use your functions to
   // - remove all classes
   // - then add the class name: "gray"
+  removeAllColors();
+  addColor("gray");
 });
 
 // Write your two functions below:
+
+function removeAllColors() {
+  box.classList.remove("red", "blue", "green", "gray");
```

Comment:
```suggestion
  box.setAttribute("class", "");
```
that is another solution I used in my code, but yours is also fine.
ref : [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute)